.build-test-rules.yml

```diff
@@ -1,6 +1,6 @@
 examples:
   disable:
-    - if: IDF_VERSION_MAJOR < 5 and CONFIG_NAME in ["esp32_c3_lcdkit", "esp32_s3_lcd_ev_board", "esp32_s3_usb_otg"]
+    - if: IDF_VERSION_MAJOR < 5 and CONFIG_NAME in ["esp32_c3_lcdkit", "esp32_s3_lcd_ev_board", "esp32_s3_usb_otg", "m5stack_core_s3"]
```

Comment:
Why do we disable IDF4.4 build for m5stack core s3 and at the same time we add `bsp/m5stack_core_s3/m5stack_core_s3_idf4.c` ? (same issue in `bsp/m5stack_core_s3/idf_component.yml`)

Reply:
IDF4.4 was removed, because audio is not working in IDF4.4 on this board. I forgot remove C file. Now it is removed.